release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -85,6 +85,13 @@ For more information, see xref:../getting_started/openshift-overview.adoc[Gettin
 [id="ocp-4-11-security"]
 === Security and compliance
 
+[id="ocp-4-11-security-oauth-server-audit-log"]
+==== OAuth server events are now logged in the audit logs
```

Comment:
Maybe worth flipping to avoid passive?

```suggestion
==== Audit logs include OAuth server events
```

Reply:
Nit: I don't know how specific we need to be, but technically we enabled audit logging for the oauth-server. So it would be more like

> OAuth server's audit events are now logged in the audit logs.